content/02.introduction.md

```diff
@@ -60,7 +61,10 @@ The dsRNA genome is then transcribed and replicated to create viral mRNAs and ne
 
 #### Transmission
 In general, respiratory viruses like coronavirus can have multiple routes of person-to-person transmission including droplet transmission (i.e. inhalation for cough, sneeze), aerosol transmission (i.e. virus suspended in air), and contact transmission (i.e. contact with oral, nasal, and eye mucous membranes).
+Other modes of transmission such as through touching surfaces or objects and then touching mucous membranes can also be investigated.
 While there does not appear to be experimental evidence to conclude which mode of transmission different coronaviruses use [@doi:10.1038/s41368-020-0075-9], it has been suggested that the coronavirus person-to-person transmission is droplet-based using simulations [@doi:10.1056/NEJMoa032867] and trace studies [@doi:10.3390/ijerph15122757].
+Other aspects of transmission to investigate is the relationship of infectiousness or virus shedding to disease period or symptoms and also which types of transmission events are commonly responsible for cases (such as transmission between relatives, nosocomial transmissions, and other possible types of interactions). Some information about these aspects is available for the highly pathogenic coronaviruses SARS (or SARS-CoV-1) and MERS [@doi:10.1016/B978-0-12-385885-6.00009-2][@doi:10.1038/nrmicro.2016.81].
```

Comment:
```suggestion
Other aspects of transmission to investigate are the relationship between infectiousness and virus shedding with disease period or symptoms and also the frequencies of various types of transmission events, such as transmission between relatives, nosocomial transmissions, and other possible types of interactions.
Some information about these characteristics of transmission are available for the highly pathogenic coronaviruses SARS (or SARS-CoV-1) and MERS [@doi:10.1016/B978-0-12-385885-6.00009-2][@doi:10.1038/nrmicro.2016.81].
```

Reply:
The official terminology is SARS-CoV and MERS-CoV, isn't it?

Reply:
I think that makes sense @matfax. Just looking at the abstract and intro of the Nature Reviews Microbiology 2016 review referenced, they used SARS-CoV and MERS-CoV when talking about the virus (and SARS and MERS when just talking about the disease).
This means we need to correct SARS to SARS-CoV (and leave the parentheses SARS-CoV-1 which I have also seen people use in recent papers, probably to distinguish from -2) and MERS to MERS-CoV in the excerpt above.

Reply:
@rando2 @matfax I'm not sure if the "frequency" correction explains the point clearly enough although it definitely reads better than what I initially tried. How about something like:
"... and also the proportions of cases that are attributable to various types of transmission events, such as ..."
& keep the other changes you made in the sentence
& add the terminology corrections "SARS-CoV (or SARS-CoV-1)" and "MERS-CoV"
I can try to fix this myself but I don't want to make a learner's mistake. @rando2 would you fix this one? (& then I can "commit suggestion" if needed)

Reply:
@yemarshall done! And I don't think this is the first time we talk about SARS-CoV so we can just make sure to note at the beginning that it is sometimes designated as SARS-CoV-1 to distinguish it from SARS-CoV-2.